app/views/errors/internal_server_error.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= title('Internal Server Error (500)') %>
+
+<p class="text-base text-gold-600">500</p>
+<h1 class="mt-4 text-5xl text-slate-900 font-bold">Server error</h1>
+<p class="leading-7 mt-6 text-slate-600">We're sorry, but something went wrong.</p>
+<div class="mt-10 flex items-center justify-center gap-x-6">
+    <a href="/home" class="rounded-md bg-teal-700 px-3.5 py-2.5 text-sm text-white shadow-sm hover:bg-teal-800 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-teal-600">Go back home</a>
```

Comment:
Nit: We can make a couple of QOL improvements to this now that all of Rails/ Tailwind is available to us: 
* We can use a dynamic link_to here, it will take users to their dashboard if logged in / the home page if not. 
* We can also use our button styling here too.
* This will need updated in the 422 page too

```suggestion
    <%= link_to "Go back home", root_path, class: "button button--primary" %>
```